### ISSUE TITLE
Bundle: Error messages and alert accessibility

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2928]
### Story

As a assistive technology enabled user
I want errors preventing me from moving forward to be clearly called out
so that I can fix the error without confusion

Our error alerts and fields fail the following WCAG criteria today:
- [4.1.2 Name, Role, Value:](https://www.w3.org/TR/WCAG20/#ensure-compat-rsv)
- [1.4.1 Use of Color](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-without-color)

### Acceptance Criteria

- [ ] #2943
- [ ] #2964
- [ ] Be sure to check all pages, including the review page on request, and check using both the org model flag an non org model flag.

### Additional Context

This story resolves issue 7 and 8 from the Trusted Tester 508 report and [remediation plan](https://docs.google.com/document/d/1L7TW2RdxAFUrWWLSw3IUYcy4Zu_1OnBl/edit).

### Issue Links
- Relates to #2913 

### Prioritization Comments
**WCAG Conformance Level:** 
**WCAG Standard that applies:** 
**Critical Failure if Not Met?:**  
**Prioritization:** 
**Reason:**

**Comment by abroddrick** [comment]
@Katherine-Osos mind explaining why @kimberly-a is assigned, is she reviewing this ticket and we should hold off on fixing it until then?

**Comment by Katherine-Osos** [comment]
@abroddrick It was a mistake. Just unassigned.

**Comment by gabydisarli** [comment]
@CocoByte It looks like you completed the work for this bundle which includes tickets 2943 and 2964, but those two tickets are open. They look done to me, should they be closed? cc: @Katherine-Osos

**Comment by Katherine-MN** [comment]
Removed "skipped design" to reflect that design documentation has already been updated to reflect these changes.
